src/module/actor/character/document.ts

```diff
@@ -369,15 +369,17 @@ class CharacterPF2e<TParent extends TokenDocumentPF2e | null = TokenDocumentPF2e
         const { resources } = this.system;
         resources.heroPoints.max = 3;
         resources.investiture = { value: 0, max: 10 };
-
         resources.focus = {
             value: resources.focus?.value || 0,
             max: 0,
             cap: 3,
         };
-
         resources.crafting = fu.mergeObject({ infusedReagents: { value: 0, max: 0 } }, resources.crafting ?? {});
         resources.crafting.infusedReagents.max = 0;
+        resources.mythicPoints = {
+            value: resources.mythicPoints?.value ?? 0,
+            max: this.itemTypes.feat.some((f) => f.system.traits.value.includes("calling")) ? 3 : 0,
```

Comment:
Can we check the (cached) setting here?

Reply:
Sure! We may want to keep mythic points working on npcs in non-mythic games, but it makes perfect sense for pcs.